bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -134,6 +134,8 @@
    public static final int             DB_LENGTH_WEATHER                 = 1000;
    public static final int             DB_LENGTH_WEATHER_CLOUDS          = 255;
 
+   public static final int             DB_LENGTH_POWER_DATA_SOURCE       = 255;
```

Comment:
@wolfgang-ch 
You said this "I would create the field power_DataSource with variable string length" but I don't know how to specify a variable length in Java. Is 255 ok ?